tests/unit/test_adapter.py

```diff
@@ -6,7 +6,7 @@
 import agate
 import boto3
 import pytest
-from moto import mock_athena, mock_glue, mock_s3
+from moto import mock_athena, mock_glue, mock_s3, mock_sts
```

Comment:
I am not sure to get why `sts` is now mandatory ?

Reply:
we can keep CatalogId parameter null for all related api calls and rely on the default AWS API behavior. However it seems that getting catalogId with sts and putting makes work with catalog_id attribute more clear. WDYT does it make sense?